content/cucumber/configuration.md

```diff
@@ -1,13 +1,75 @@
 ---
 title: Cucumber Configuration
-subtitle: cucumber.yml, environment variables
+subtitle: cucumber.yml, type registry
 polyglot: true
 ---
 
+
+# Type Registry Configuration
+
+{{% block "java" %}}
+The type registry is used to configure parameter and data table types. It can be configured by placing an implementation 
+of `cucumber.api.TypeRegistryConfigurer` on the glue path.
+
+
+```java
+public class TypeRegistryConfiguration implements TypeRegistryConfigurer {
```

Comment:
Really appreciate having an example here. I feel it could use a little explanation though. Let me know if you want some help with that ;)